datalad/support/tests/test_gitrepo.py

```diff
@@ -414,10 +414,10 @@ def test_GitRepo_fetch(test_path, orig_path, clone_path):
 
     fetched = clone.fetch(remote='origin')
     # test FetchInfo list returned by fetch
-    eq_([u'origin/master', u'origin/new_branch'],
+    eq_([u'origin/' + clone.get_active_branch(), u'origin/new_branch'],
         [commit.name for commit in fetched])
 
-    ok_clean_git(clone.path, annex=False)
+    ok_clean_git(clone.path)
```

Comment:
This is not, what was done before. `ok_clean_git` never did that. It just checked for the existence of `.git/annex` if `annex` was True. Nothing more.

But more important: The test is running on all testrepos. So it might be an annex. It's not important whether annex commands were run, but `ok_clean_git` needs to know it is allowed to be an annex, cause this might change, what that helper has to do. (direct mode and V6)